package.json

```diff
@@ -90,6 +93,7 @@
     "@typescript-eslint/parser": "^5.37.0",
     "babel-jest": "^29.2.1",
     "eslint": "^8.19.0",
+    "glob": "^9.3.0",
```

Comment:
is this a peer dep of the formatjs cli?

Reply:
It used by the `build-translations` script!